deployment/base/scripts/insertDefaults.php

```diff
@@ -37,8 +37,9 @@ function handleDirectory($dirName)
 	while (false !== ($fileName = $dir->read()))
 	{
 		$filePath = realpath("$dirName/$fileName");
-		if($fileName[0] == '.' || is_dir($filePath) || !preg_match('/^\d+\.\w+\.ini$/', $fileName))
+		if($fileName[0] == '.' || is_dir($filePath)|| !preg_match('/^\d+\.\w+?\.?\w+\.ini$/', $fileName) || preg_match('/.*\.template\.ini$/',$fileName)){
```

Comment:
@jessp01, @erankor, it won't work because we use the file name to decide on the object type.
Please fix the file name to [order].[objectType].template.ini.
You can leave the template in because it's replaced by .ini during the installation and for that exact reason you shouldn't scan it during the insertion.